README.md

```diff
@@ -218,6 +222,60 @@ the docker integration tests, run the following:
 $ sudo -E PATH=$PATH make docker
 ```
 
+A list of available test suite `make` targets can be found by running the
+following:
+
+```
+$ make help
+```
+
+### Running subsets of tests
+
+Individual tests or subsets of tests can be selected to be run. The method of
+test selection depends on which type of test framework the test is written
+with. Most of the Kata Containers test suites are written
+using either [BATS](https://github.com/sstephenson/bats) files or with
+[Ginkgo](https://github.com/onsi/ginkgo).
+
+#### Running `BATS` based tests
+
+The `BATS` based tests are shell scripts, starting with the line:
+
+```sh
+#!/usr/bin/env bats
+```
+
+This allows the `BATS` files to be executed directly.  Before executing the file,
+ensure you have `BATS` installed. The `BATS` files should be executed
+from the root directory of the tests repository to ensure they can locate all other
+necessary components. An example of how a `BATS` test is run from the `Makefile`
+looks like:
+
+```makefile
+kubernetes:
+        bash -f .ci/install_bats.sh
+        bash -f integration/kubernetes/run_kubernetes_tests.sh
+```
+
+#### Running `Ginkgo` based tests
+
+`Ginkgo` supports selecting, filtering and excluding tests to be run using command
+line parameters. The tests repository `Makefile` supports passing the environment
+variables `FOCUS` and `SKIP` through to the `ginkgo` command in most cases. Check
+the `Makefile` specifics before you use this functionality.
+
+`ginkgo` accepts [`Golang` `regexp`](https://golang.org/pkg/regexp/) regular expressions
```

Comment:
Again, you can write "Golang" and "regex" here as they will be accepted by the spell-checker.